package.json

```diff
@@ -7,7 +7,7 @@
     "build": "tsc && cp -r views dist/",
     "devstart": "nodemon",
     "test": "jest --forceExit",
-    "lint": "tslint -p .",
+    "lint": "tslint '{src,test}/**/*'",
```

Comment:
Specify the directories to analyse rather than use the settings in the tsconfig.json file (which deliberately doesn't write generated test JS files to the 'dist' directory).